tests/app_engine/resources/k8s_handler.py

```diff
@@ -53,3 +53,42 @@ def remove_clusterrole_binding(project_id, cluster_id, zone, shipper_name, crb_n
     except Exception as e:
         print(f"\n{shipper_name} cluster role binding deletion has failed, please manually delete {crb_name}:\n{e}")
         print(f"kubectl delete clusterrolebinding {crb_name}")
+
+
+def list_namespace_deployments(project_id, cluster_id, zone, namespace, keywords=None):
+    apis_api = client.AppsV1Api(api_client=K8SApiClient(project_id, zone, cluster_id))
+    resp = apis_api.list_namespaced_deployment(namespace=namespace)
+    if keywords:
+        deps = [dep.metadata.name for dep in resp.items if any(ele in dep.metadata.name for ele in keywords)]
+    else:
+        deps = [dep.metadata.name for dep in resp.items]
+    return deps
+
+
+def remove_deployments_in_namespace(project_id, cluster_id, zone, namespace, deps=None, keywords=None):
+    """
+    remove deployments in namespace
+    :param project_id:
+    :param cluster_id:
+    :param zone:
+    :param namespace: string, namespace name
+    :param deps: list, a list of all deployments to be deleted under the given namespace
+    :param keywords: list, in case deps wasn't supplied, all deployments who's in their name one of the items in keyword
+                        will be listed for deletion
+    :return:
+    """
+    if not deps:
+        deps = list_namespace_deployments(project_id, cluster_id, zone, namespace, keywords)
```

Comment:
if keywords is `None` we get all deps?

Reply:
and you didn't mention which deployment you'd like to delete